### ISSUE TITLE
Smak og Smakssans

### ISSUE THREAD

**Issue by kaalrust** [realfagstermer/realfagstermer#640]
[Smakssans](https://app.uio.no/ub/emnesok/realfagstermer/search?term=Smakssans) 
[Smak](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c008442)

I Humord brukes "Smak" om estetisk sans, men jeg kan ikke se noen forskjell i bruken for disse to?

**Comment by BioHeidi** [comment]
Vi må vel slå de sammen, selv om vi da mister den legendariske strengen Smak : Insekter
@danmichaelo @kyrretl :smile_cat:

**Comment by Hypsibius** [comment]
Som smakstilsetning eller som fashionistas?

**Comment by danmichaelo** [comment]
Mangler vi virkelig Smak : Insekter : 1700-tallet : Sovjetunionen ?? Skuffende :/

**Comment by danmichaelo** [comment]
Hadde vi både `Lukt` og `Luktesans` også..? Ja, sannelig!

Med eller uten "sans" i foretrukket term?

**Comment by Hypsibius** [comment]
Jo, men blir ikkje det som forskjellen mellom 'lyd' og 'hørsel'?

**Comment by Hypsibius** [comment]
Altså, forskjellen mellom signal og evne til å oppfange det? Medan 'smak' i tillegg har den overførte tydinga 'estetisk sans' og derfor kan mistydast (endå lenger)

**Comment by BioHeidi** [comment]
Her trengs det litt rydding. Vi kan fordele dokumentene som har fått Lukt over på Luktesans, Luktstoffer, Feromoner
Slå sammen Luktstoffer og Duftstoffer
Slå sammen Luktesans og Lukteorganer

Noe mer?

**Comment by Hypsibius** [comment]
Har vi døme på smak, som i preferanse?
